src/app/public/theming/theme.service.ts

```diff
@@ -125,10 +125,14 @@ export class SkyThemeService {
   }
 
   private addHostClass(className: string): void {
-    this.renderer.addClass(this.hostEl, className);
+    if (this.renderer && this.hostEl) {
```

Comment:
I didn't catch this on my initial pass. Putting the `if` around these statements makes calling `init()` optional, but `init()` should always be called by the component that provides the service. I'm guessing this was done to work around errors in the unit tests since `init()` is not called there; I would suggest either calling `init()` in the unit tests or using a mock service in place of `SkyThemeService`.

Reply:
Here's an example of using a mock theme service: https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/input-box/input-box.component.spec.ts#L274